modules/caffeine/src/test/scala/scalacache/caffeine/CaffeineCacheSpec.scala

```diff
@@ -114,4 +127,24 @@ class CaffeineCacheSpec extends AnyFlatSpec with Matchers with BeforeAndAfter wi
       IO(underlying.getIfPresent("key1")).map(_ shouldBe null)
   }
 
+  behavior of "get after put"
+
+  it should "store the given key-value pair in the underlying cache with no TTL, then get it back" in ticked { _ =>
+    val underlying = newCCache
+    val cache      = newIOCache(underlying)
+    cache.put("key1")("hello", None) *>
+      cache.get("key1").map { _ shouldBe defined }
+  }
+
+  behavior of "get put with TTL"
+
+  it should "store the given key-value pair in the underlying cache with the given TTL, then get it back when not expired" in ticked {
```

Comment:
Let's have another one like this, but with an `IO.sleep` between put and get. It'll be instant because of how `ticked` works.

Reply:
Added a couple of tests with sleep.